draft-ietf-quic-qlog-main-schema.md

```diff
@@ -1873,17 +1873,88 @@ still provide adequate output for incomplete logs.
+  identify individual connections, endpoints and potentially users.
+
+* QUIC Connection IDs and contents of NEW_CONNECTION_ID frames, which can be
+  used to identify and track users across geographical networks {{!RFC9000}}.
+
+* Accurate event timestamps, event counts, packet and frame sizes, which can be
+  used with other data sources (e.g., encrypted packet captures) to correlate
+  qlog data to a specific connection or user.
+
+* Token values, which can contain address validation details and previous
+  connection information which could allow correlation of multiple connections.
+
+* TLS (decryption) keys, which can be used with other data sources (e.g.,
+  encrypted packet captures) to correlate qlog data to a specific connection or
+  user or leak additional information.
+
+* Full or partial, encrypted or plaintext raw packet and frame payloads (e.g.,
+  HTTP Field values, HTTP response data, TLS SNI field values), which can
+  contain other sensitive information. In qlog, these are typically (but not
+  exclusively) contained in fields of the RawInfo type (see {{raw-info}}). qlog
+  users should be particularly hesitant to include this information in all but
+  the most stringent conditions.
+
+The simplest and most extreme form of protection against abuse of this
+information is the complete deletion of a given field, which is equivalent to
+not logging the field(s) in question. While deletion completely protects the
+data in the deleted fields from the risk of compromise, it also reduces the
+utility of the dataset as a whole. As such, a balance should be found between
+logging these fields and the potential risks inherent in their (involuntary)
+disclosure. This balance depends on the use case at hand (e.g., research
+datasets have different requirements to live operational troubleshooting). As
+such, operators SHOULD only capture the minimal subset of qlog data relevant to
+their specific purpose. Consequently, qlog implementers SHOULD allow for
+changing this subset in a dynamic fashion, ideally even on a per-connection
+basis.
+
+For the remaining fields deemed necessary for the use case at hand, as per
+{{!RFC6973}}, operators must be aware that the data will be at risk of
+compromise. As such, measures should be taken to firstly reduce the risk of
+compromise and secondly reduce the risk of abuse of compromised data. While a
+full discussion of both aspects is out of scope for this document, the following
+paragraphs discuss high level concepts that can be applied to qlog data.
+
+To reduce the risk of compromise, operators can take measures such as: limiting
+the amount of time data is stored, encrypting data at rest, limiting access
+rights to the data, and auditing data usage practices. qlog deployments SHOULD
+have integrated options for easy qlog data deletion or (aggressive) aggregation.
+
+To reduce the risk of data abuse after compromise, sensitive fields can be
```

Comment:
does it have to be just "sensitive fields"? Just saying data fields would likely hold true.